src/main/java/Parser.java

```diff
@@ -0,0 +1,44 @@
+public class Parser {
+    private String userInput;
+    private String searchQuery;
+
+    public String parse(String userInput) {
+        this.userInput = userInput;
+        if (userInput.equals("list")) {                                                  //Lists out the tasks
+            return "list";
+        } else if (userInput.split("\\s")[0].equals("done")) {
```

Comment:
It might be better to change the if-else statements to a switch statement instead to improve readability.